axelrod/strategies/averagecopier.py

```diff
@@ -21,8 +22,7 @@ class AverageCopier(Player):
         'manipulates_state': False
     }
 
-    @staticmethod
-    def strategy(opponent: Player) -> Action:
```

Comment:
This was being pointed out as an error by `mypy` something to do with the method not being in line with the parent class.